Documents/Management/managementDocuments.md

```diff
@@ -0,0 +1,60 @@
+this document contains the main documentation for the management of the project like :
+
+- [Project charter](#project-charter)
+- [Task schedule](#task-schedule)
+- [RACI matrix](#raci-matrix)
+- [MoSCoW](#moscow)
+- [Target Key Performace Indicators](#target-key-performace-indicators)
+- [Statement of work](#statement-of-work)
+
+## Project charter
+
+The project charter is a document that formally authorizes a project. The project charter provides a high-level overview of the project and is a critical document for getting the project started. Our project charter is [here](projectCharter.md).
+
+## Task schedule
+
+The task schedule of this project is on the projects on GitHub [here]( https://github.com/orgs/algosup/projects/25/views/1).
+
+## RACI matrix
+
+|Tasks|Project Manager|Program Manager|Technical Leader|Software Engineer|Quality Assurance|Technical Writer|Customer|
+|-----|---------------|---------------|----------------|-----------------|------------------|-----------------|--------|
+|Project brief|I|I|I|I|I|I|R / A|
+|Project charter|R / A|C / I|C/I|C / I|C / I|C / I|C / I|
+|Task schedule|R / A|C|C / I|C / I|C / I|C / I|I|
+|Code|I|I|C|R / A|C|C|X|
+|Code review|C / I|C / I|R / A|R|C / I|C / I|X|
+|Code documentation|C / I|I|C|R / A|C|C|X|
+|Test planning|C / I|I|A|C / I|R / A|C|X|
+|Test execution|C / I|I|I|C / I|R / A|C|X|
+|User manual|C / I|I|C|C|C|R / A|I|
+
+Legend:
+
+|Letter|Name|Description|
+|------|----|-----------|
+|R|Responsible|The person who does the work to achieve the task. There should be exactly one person with this assignment for each task.|
+|A|Accountable|The person who is accountable for the correct and thorough completion of the task.|
+|C|Consulted|The people who provide information for the project.|
+|I|Informed|The people who are kept informed of the project's progress.|
+|X|Not applicable|The task is not applicable to the person or role.|
+
+## MoSCoW
+
+| Must Have| Should Have | Could Have | Won't Have |
+| -------- | ----------- | ---------- | ---------- |
+| - Battery lifespan of seven days <br> - Manage all security issues <br> - Shock detection <br> - Unlock the device | - Unlock the device with NFC <br> - Shock notification|X| - GNSS position acquisition <br> - Activate/Desactivate by app|
```

Comment:
Theoretically, yes, we will be able to activate and deactivate the hardware via the application. The application uses Bluetooth, and we're working on Bluetooth. We'll provide the customer with a document at the end to explain whether we've changed what they had done, and how to implement it in their application (without going into detail, just specifying the name of each new function, and its use in the application).